deps/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/deps/Helpers.java

```diff
@@ -119,23 +118,23 @@ else if(expectedValue instanceof Map)
                     {
                         if(message == null)
                         {
-                            assertTrue("Map contains invalid Object", false);
+                            fail("Map contains invalid Object");
                         }
                         else
                         {
-                            assertTrue(message, false);
+                            fail(message);
                         }
                     }
                 }
                 else if(expectedValue instanceof ArrayList)
                 {
                     if(message == null)
                     {
-                        assertTrue("Map failed on " + key + ": " + actualValue + " != " + expectedValue, actualValue.toString().equals(expectedValue.toString()));
+                        Assert.assertEquals("Map failed on " + key + ": " + actualValue + " != " + expectedValue, actualValue.toString(), expectedValue.toString());
```

Comment:
do we need to state the class qualifier here?

Reply:
Nope. Both objects here are strings, so the assertEquals(string, string) overload is called

Reply:
No, what I meant is, why do we need to call `Assert.assertEquals` (and not `assertEquals` directly).